packages/manager/apps/dedicated/client/app/billing/autoRenew/agreements/details/details.routing.js

```diff
@@ -11,6 +13,10 @@ export default /* @ngInject */ (
       template,
       controller: 'UserAccount.controllers.agreements.details',
       controllerAs: 'ctrl',
+      resolve: {
+        agreementId: /* @ngInject */ ($transition$) =>
+          toInteger($transition$.params().id),
```

Comment:
Maybe we could use type int for param id instead https://ui-router.github.io/ng1/docs/latest/classes/params.paramtypes.html

Reply:
Thanks for the link. It was useful. This is done :)